presto-release-tools/src/test/resources/release-notes-test/release-0.231_expected.rst

```diff
@@ -2,6 +2,12 @@
 Release 0.231
 =============
 
+**Highlights**
+==============
+
+**Details**
+==============
```

Comment:
The number of "=" need to match the length of the last line, so we need 3 less "=".

Reply:
done